src/main/java/bio/terra/service/dataset/flight/create/CreateDatasetMetadataStep.java

```diff
@@ -35,16 +35,17 @@ public CreateDatasetMetadataStep(DatasetDao datasetDao, DatasetRequestModel data
     @Override
     public StepResult doStep(FlightContext context) {
         try {
-            FlightMap workingMap = context.getWorkingMap();
-            UUID projectResourceId = workingMap.get(DatasetWorkingMapKeys.PROJECT_RESOURCE_ID, UUID.class);
-            UUID datasetId = workingMap.get(DatasetWorkingMapKeys.DATASET_ID, UUID.class);
-            Dataset newDataset = DatasetUtils.convertRequestWithGeneratedNames(datasetRequest)
+            final FlightMap workingMap = context.getWorkingMap();
```

Comment:
Was the only change in this file to add `final`? That doesn't seem worth changing a file for, if nothing else in it was changed.

Reply:
You and @nmalfroy can duke it out https://github.com/DataBiosphere/jade-data-repo/pull/901#discussion_r623531297

Reply:
It looks like there were other changes too, no?  If it is the only change, I'm ok with leaving it for now